cinema_service/settings.py

```
@@ -20,12 +25,10 @@
 # See https://docs.djangoproject.com/en/4.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = (
-    "django-insecure-6vubhk2$++agnctay_4pxy_8cq)mosmn(*-#2b^v4cgsh-^!i3"
-)
+SECRET_KEY = os.environ["SECRET_KEY"]
```

SECRET_KEY is read with `os.environ["SECRET_KEY"]` which will raise KeyError if not set. This may be desirable for production (fail fast), but consider raising a clearer `django.core.exceptions.ImproperlyConfigured` error or documenting that the env var is required. Alternatively, for local/dev only, you can provide a clearly marked insecure fallback (but do not use that in production).